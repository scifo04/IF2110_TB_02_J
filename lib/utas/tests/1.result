lib/utas/tests/1.in lib/utas/tests/1.out: TRUE
