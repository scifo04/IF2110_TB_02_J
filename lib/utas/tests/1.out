31/10/2023 23:59:59
ABC
1
